opentelemetry_collector/receiver/dockerstats/scraper.go

```diff
@@ -109,31 +134,26 @@ func (s *scraper) export() {
 		}
 		labelValues := []*mpb.LabelValue{metricgenerator.MakeLabelValue(name)}
 
-		stats, err := s.readStats(ctx, container.ID)
+		stats, err := s.readResourceUsageStats(ctx, container.ID)
 		if err != nil {
-			glog.Warningf("readStats failed for container %s: %v", container.ID, err)
-			continue
+			glog.Warningf("readStats failed for container %s(%s): %v", name, container.ID, err)
```

Comment:
In general OpenTelemetry Collector uses the go.uber.org/zap logger. One is passed into the Factory's CreateMetricsReceiver function as part of ReceiverCreateParams and can be passed into the scraper from there.

Reply:
Requires a little bit of plumbing to use zap. Filed https://github.com/GoogleCloudPlatform/appengine-sidecars-docker/issues/147 and I'll address it in a separate pull request.